linux-anvil/entrypoint

```diff
@@ -1,7 +1,15 @@
 #!/bin/bash -il
 
+# Create conda user with the same uid as the host, so the container can write
+# to mounted volumes
+# Adapted from https://denibertovic.com/posts/handling-permissions-with-docker-volumes/
+USER_ID=${HOST_USER_ID:-9001}
+useradd --shell /bin/bash -u $USER_ID -o -c "" -m conda
+export HOME=/home/conda
+chown -R conda /opt/conda
+
 # Source everything that needs to be.
 . /opt/docker/bin/entrypoint_source
 
 # Run whatever the user wants.
-exec "$@"
+exec /usr/local/bin/gosu conda "$@"
```

Comment:
So this is a bit silly, but should be easy to do. Can we set `USER` and `LOGNAME` explicitly? It appears that `gosu` is not doing this for us. As a result, they are both `root` ATM.